py2puml/inspection/inspectnamedtuple.py

```diff
@@ -16,5 +16,6 @@ def inspect_namedtuple_type(
         attributes=[
             UmlAttribute(tuple_field, 'Any', False)
             for tuple_field in namedtuple_type._fields
-        ]
+        ],
+        methods=[],
```

Comment:
technically they can have methods. should we support it?

Reply:
let's see after I figure it out for regular classes and then we get back to this one to do it or split it.